<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HlavneOkno</class>
 <widget class="QMainWindow" name="HlavneOkno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HlavneOkno</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="PracovnaPlocha" name="pracovnaPlocha" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuS_bor">
    <property name="title">
     <string>Súbor</string>
    </property>
    <addaction name="actionNov_dr_ha"/>
    <addaction name="actionOtvori_existuj_cu_dr_hu"/>
    <addaction name="separator"/>
    <addaction name="actionUlo_i"/>
    <addaction name="actionUlo_i_ako"/>
    <addaction name="separator"/>
    <addaction name="actionUkon_i"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionExport_do_PDF"/>
    <addaction name="actionExport_do_SVG"/>
   </widget>
   <addaction name="menuS_bor"/>
   <addaction name="menuExport"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="vlastnostiDokumentuWidget">
   <property name="windowTitle">
    <string>Vlastnosti dokumentu</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="PanelVlastnosti" name="vlastnostiDokumentu" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="vlastnostiObjektuWidget">
   <property name="windowTitle">
    <string>Vlastnosti objektu</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="PanelVlastnosti" name="vlastnostiObjektu" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Kontroly</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="KontrolyPanel" name="kontroly"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Chyby</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="ChybyPanel" name="chyby"/>
  </widget>
  <action name="actionNov_dr_ha">
   <property name="text">
    <string>Nová dráha</string>
   </property>
  </action>
  <action name="actionOtvori_existuj_cu_dr_hu">
   <property name="text">
    <string>Otvoriť existujúcu dráhu</string>
   </property>
  </action>
  <action name="actionUlo_i">
   <property name="text">
    <string>Uložiť</string>
   </property>
  </action>
  <action name="actionUlo_i_ako">
   <property name="text">
    <string>Uložiť ako</string>
   </property>
  </action>
  <action name="actionUkon_i">
   <property name="text">
    <string>Ukončiť</string>
   </property>
  </action>
  <action name="actionExport_do_PDF">
   <property name="text">
    <string>Export do PDF</string>
   </property>
  </action>
  <action name="actionExport_do_SVG">
   <property name="text">
    <string>Export do SVG</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PracovnaPlocha</class>
   <extends>QWidget</extends>
   <header>pracovnaplocha.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanelVlastnosti</class>
   <extends>QWidget</extends>
   <header location="global">panelvlastnosti.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KontrolyPanel</class>
   <extends>QWidget</extends>
   <header>kontrolypanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChybyPanel</class>
   <extends>QWidget</extends>
   <header>chybypanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
